components/eventing-controller/controllers/subscriptionv2/jetstream/test_utils_test.go

```diff
@@ -0,0 +1,584 @@
+package jetstream_test
```

Comment:
test_utils_test? this means its a test for test_utils?

Reply:
test_utils - because these are test_utils, and _test at the end, because it is in the jetstream_test package

Reply:
Let's address this in a [follow-up](https://github.com/kyma-project/kyma/issues/16618)